content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -166,6 +168,8 @@ spec:
         value: false
 ```
 
+#### Learn more Learn more about .NET auto-instrumentation
```

Comment:
Copy-paste error here: "Learn more Learn more". In any case, I'd suggest keeping the heading titles short (here and elsewhere in the page):

```suggestion
#### Learn more
```

It's clear from the line just below the heading, what the "more" is.

Reply:
Yes, but the markdown linter started screaming at me because the same header was duplicated for each section. If I restore to what it was, can we bypass it?